src/cl/op-node/op_node_launcher.star

```diff
@@ -188,10 +188,15 @@ def get_beacon_config(
         "--altda.da-server=" + da_server_context.http_url,
     ]
 
+    supervisor_params = _filter.first(supervisors_params)
+
     # configure files
 
     files = {
-        ethereum_package_constants.GENESIS_DATA_MOUNTPOINT_ON_CLIENTS: launcher.deployment_output,
+        ethereum_package_constants.GENESIS_DATA_MOUNTPOINT_ON_CLIENTS: Directory(
+            launcher.deployment_output,
+            supervisor_params.superchain.dependency_set.name,
```

Comment:
suprrvisor_params can be null here, you'll need a check before you access its properties